src/renderer/views/Watch/Watch.js

```diff
@@ -274,8 +274,13 @@ export default Vue.extend({
 
             throw new Error(`${reason}: ${subReason}`)
           }
-
-          this.videoTitle = result.videoDetails.title
+          try {
+            // workaround for title localization
+            this.videoTitle = result.response.contents.twoColumnWatchNextResults.results.results.contents[0].videoPrimaryInfoRenderer.title.runs[0].text
+          } catch {
```

Comment:
It's a good idea to catch and log the error, that way it can be fixed if it breaks in the future.
```suggestion
          } catch (err) {
            console.error('Failed to extract localised title, falling back to the standard one.', err)
```